core/dive.c

```diff
@@ -3118,6 +3154,7 @@ struct dive *clone_delete_divecomputer(const struct dive *d, int dc_number)
 	res->id = dive_getUniqID();
 
 	delete_divecomputer(res, dc_number);
+	force_fixup_dive(res);
```

Comment:
I'm curious why this is needed here...

Reply:
This is a bug I discovered on this way: If you delta a dive computer, the maxdepth etc is not recalculated for the divelist.